oak_enclave_runtime_support/src/libm.rs

```diff
@@ -25,3 +25,75 @@ pub extern "C" fn fmodf(a: f32, b: f32) -> f32 {
 pub extern "C" fn fmod(a: f64, b: f64) -> f64 {
     libm::fmod(a, b)
 }
+
+// These functions are required by oak_tensorflow_enclave.
+
+#[no_mangle]
+pub extern "C" fn sqrtf(x: f32) -> f32 {
+    libm::sqrtf(x)
```

Comment:
Is this the same libm that comes from here: https://github.com/project-oak/oak/tree/main/cc/libm
Or it's a rust crate?

Reply:
No, it's a Rust crate.

Reply:
Got it
So https://github.com/project-oak/oak/tree/main/cc/libm is no longer needed, right?

Reply:
I think it still is, but I'm not sure.